CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+## Contributing to Axelar Examples
+
+We appreciate your interest in contributing to Axelar Examples! This document `CONTRIBUTING.md` provides a high-level overview of how you can get involved.
+
+### Table of Contents
+  * [Introduction](#introduction)
+  * [Getting Started](#getting-started)
+  * [Issues](#issues)
+  * [Pull Requests](#pull-requests)
+  * [Code of Conduct](#code-of-conduct)
+
+### Introduction
+
+`Axelar Examples` is a collection of examples and tutorials for the Axelar Network. Before you start, we encourage you to read the rest of this document, which contains information on how and what you can contribute.
+
+### Getting Started
+
+If you are new to the Axelar Network, we encourage you to read the [Axelar Documentation](https://docs.axelar.network/) to learn more about the Axelar Network and how to get started. 
+
+If you are new to contributing to open-source projects, we encourage you to read [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/). If you are new to contributing to Axelar, we encourage you to read [`CONTRIBUTING.md`](./CONTRIBUTING.md) in the Axelar examples repository.
+
+
+Before you start contributing, you need to:
+
+1. Fork the repository and clone it to your local machine.
+2. Create a new branch for your contribution.
```

Comment:
Is this a required step? Open source contributors won't have permission to push to our repo anyway, so forking effectively is a new branch.

Reply:
Yes, they won't have permission to push to the org repository. That's why I suggested a fork.